ctx.py

```diff
@@ -203,11 +203,13 @@ def cfg_factory():
     return factory
 
 
-def configure_default_logging(stdout_level=None):
+def configure_default_logging(stdout_level=None, whd_log_file_path=None):
```

Comment:
this is a general-purpose-function for all of cc-utils / gardener-cicd. I would not like to add a special-case-argument for whd, tbh

I understand you want to re-use the existing code. How about splitting this function into two parts:
1. one that will create / return a cfg-dict (identical to the one that is already created in this function)
2. one that will by default call the first function, and configure the logging frmwk

for WHD, you could then call the first function, enrich the default-dict with additional cfg and then "manually" configure the logging-module. Maybe the second function is actually not that much of a value-add, and might be omitted (need to check existing usages to tell)

Reply:
as an alternative, I would also not be opposed to creating a dedicated logging-cfg just for WHD. WHD is a special, separate component (and should actually not necessarily live within cc-utils anyway), so why not have it define a custom logging-cfg?

Reply:
as an alternative, thinking some more about it. if we maybe named this path "logfile" or sth similarly generic, it would make some sense to add it here